docs/quick-start.md

```diff
@@ -25,7 +25,7 @@ We suggest using this `yy` shell wrapper that provides the ability to change the
 
 ```bash
 function yy() {
-	local tmp="$(mktemp -t "yazi-cwd.XXXXXX")"
+	local tmp="$(mktemp -t "yazi-cwd.XXXXXX")" cwd
```

Comment:
I don't quite understand its purpose. Can you explain it?

Reply:
your function creates global variable `cwd`, my change scopes it to function `yy`. read the whole file to understand context. (you create variable cwd on line 30).

there's no work to do. the merge request is complete.

Reply:
Ah I see now, reopened